includes/core/classes/class-settings.php

```diff
@@ -342,8 +449,8 @@ public function get_name_field( string $sub_page, string $section, string $optio
 	 * @return array
 	 */
 	public function get_sub_pages(): array {
-		$sub_pages = array(
-			'general'    => array(
+		$gp_admin_tabs = array(
```

Comment:
maybe `$contents`? (This is more than just tabs). Also, don't need `$gp` here since local variable so I would drop it.